CHANGES.rst

```diff
@@ -1,3 +1,13 @@
+0.9.3
+----------------
+* Fix parsing of SecurityMethod in capabilities
```

Comment:
From now on it would be better to add the changelog entries as part of the PRs. @astropy/pyvo-maintainers: we have an astropy bot to remind about it. It may be a bit of an overshoot for pyvo, but you're very welcome to use it.